zipf/bin/plotcounts.py

```diff
@@ -76,15 +76,18 @@ def main(args):
     ax = df.plot.scatter(x='word_frequency', y='rank', loglog=True,
                          figsize=[12, 6], grid=True, xlim=args.xlim)
 
-    alpha = get_power_law_params(df['word_frequency'].to_numpy())
+    alpha, beta = get_power_law_params(df['word_frequency'].to_numpy())
     print('alpha:', alpha)
+
     # Since the ranks are already sorted, we can take the last one instead of
     # computing which row has the highest rank
     max_rank = df['rank'].to_numpy()[-1]
+
     # Use the range of the data as the boundaries when drawing the power law curve
     curve_xmin = df['word_frequency'].min()
     curve_xmax = df['word_frequency'].max()
 
+    # Plot the result
```

Comment:
Should we be encouraging comments in Python scripts? It's better for the script to be readable without comments, right?

Reply:
I've never bought into that 100%, and I think it helps orient readers in this case.